liquibase/changeLogs/capture/tables/datumConvertedValues/datumConvertedValues.sql

```diff
@@ -0,0 +1,12 @@
+create table ${AQTS_SCHEMA_NAME}.datum_converted_values
+(datum_converted_values_id                int8 NOT NULL GENERATED BY DEFAULT AS IDENTITY,
+json_data_id                              int8 NOT NULL,
+field_visit_identifier                    text NOT NULL,
+target_datum                              text NOT NULL,
+unit                                      text NOT NULL,
+numeric_value                             numeric NOT NULL,
+display_value                             text NOT NULL,
+partition_number                          integer NOT NULL,
+primary key (datum_converted_values_id, partition_number)
+)
+PARTITION BY LIST (partition_number);
```

Comment:
Style fuss - in most of the other changesets we tend to just keep the postgres reserved words lowercase (line 12 and 2).  I don't have a strong opinion here, just pointing it out if we want to follow the established convention.